04/File.js

```diff
@@ -1,8 +1,35 @@
 import React from 'react';
-
 class File extends React.Component {
+
+    onChange= e => {
+        const {onChange} = this.props;
+        // const{getFile} = this.props;
+    
+    
+        const  file  = e.target.files[0];
+        console.log(file)
+        if(file && file.type.includes('text')) {
+            const reader = new FileReader();
+         
+          
+            reader.onload = function(readerEvent) {
```

Comment:
Ponieważ użyłaś tutaj funkcji normalnej (a nie strzałkowej) to `this` wewnątrz niej zmienił kontekst.
Dlatego `this.props.getFile()` nie działa. Wystarczy wstawić tutaj funkcję strzałkową tj. `readerEvent => { /* ... */ }`.
Mogłabyć taż przed tą funkcją zrobić destrukturyzację tj. `const {getFile} = this.props` i też problem byłby rozwiązany.